views/homepage/index.html

```diff
@@ -0,0 +1,156 @@
+              </div>
+              <div class="teach">
+                <h3>Use it to teach</h3>
+                <ul>
+                  <li>Editing HTML content</li>
+                  <li>CSS colors</li>
+                  <li>Creating an interactive element<br/> with JavaScript</li>
+                </ul>
+                <a href="{{ backToSchoolPublishedUrl }}" title="Read the Back-to-School Postcard teaching kit" class="teaching-kit">Postcard Teaching Kit</a>
+              </div>
+            </div>
+
+            <!-- Comic Strip Project -->
+            <div class="project comic-strip">
+              <h2 class="title">Current Events Comic Strip</h2>
+              <div class="thumbnail">
+                <a href="{{ comicStripRemixUrl }}" title="Remix the Current Events Comic Strip project" class="image"></a>
+              </div>
+              <p class="description">
+                Create your very own version of this comic strip.
+              </p>
+              <a href="{{ comicStripRemixUrl }}" title="Remix the Comic Strip project"  class="remix-button">Remix this Project</a>
+              <div class="or-wrapper">
+                <hr />
+                <div class="or">or</div>
+              </div>
+              <div class="teach">
+                <h3>Use it to teach</h3>
+                <ul>
+                  <li>Editing HTML content</li>
+                  <li>Finding Creative Commons-licensed images</li>
+                  <li>CSS colors</li>
+                </ul>
+                <a href="{{ comicStripPublishedUrl }}" title="Read the Comic Strip project teaching kit" class="teaching-kit">Comic Strip Teaching Kit</a>
+              </div>
+            </div>
+
+          </div>
+      </div>
+    </section>
+
+    <footer>
+      <div class="content-wrapper">
+        <div class="project-info">
+          <a class="mozilla-logo" title="Mozilla" href="https://www.mozilla.org/en-US/"><img alt="Mozilla" src="img/mozilla-logo.svg" /></a>
+          <p class="thimble-context">
+            Thimble is part of the Mozilla Learning Network. <br/>
+            Learn to teach like mozilla at <a href="http://teach.mozilla.org">teach.mozilla.org</a>.
+          </p>
+        </div>
+
+        <ul class="social-links">
+          <li><a title="Stuck? Get help!" class="help" href="https://support.mozilla.org/en-US/products/webmaker/thimble"><span class="icon"></span>Stuck? Get help!</a></li>
+          <li><a title="Talk to us on Twitter" class="twitter" href="https://twitter.com/mozteach"><span class="icon"></span>Talk to us on Twitter</a></li>
+          <li><a title="Contribute on Github" class="github" href="https://github.com/mozilla/thimble.webmaker.org"><span class="icon"></span>Contribute on Github</a></li>
+        </ul>
+      </div>
+    </footer>
+
+    <!-- Pre-fetch the main editor file while a user is on this page -->
+    <script async src="{{ editorHOST }}/dist/bramble.js"></script>
```

Comment:
This causes some havoc:

![screen shot 2015-08-27 at 12 44 14 pm](https://cloud.githubusercontent.com/assets/1616860/9526784/634a18ca-4cb9-11e5-88ea-3943d01effb0.png)